core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -573,7 +580,7 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 	//
 	// In all cases, the best thing we can do is go into a retry loop and keep
 	// trying to send the transaction over again.
-	return errors.Wrapf(sendError, "unknown error while sending transaction %s (eth_tx ID %d)", attempt.Hash.Hex(), etx.ID), true
+	return errors.Wrapf(sendError, "error while sending transaction %s (eth_tx ID %d)", attempt.Hash.Hex(), etx.ID), true
```

Comment:
nit: In error message, say retryable error while sending.
So reader knows this will be retried